src/Microsoft.AspNet.Mvc.Core/ModelBinding/BodyModelBinder.cs

```diff
@@ -26,14 +25,19 @@ public BodyModelBinder()
         }
 
         /// <inheritdoc />
-        protected async override Task<ModelBindingResult> BindModelCoreAsync([NotNull] ModelBindingContext bindingContext)
+        protected async override Task<ModelBindingResult> BindModelCoreAsync(
+            [NotNull] ModelBindingContext bindingContext)
         {
             var requestServices = bindingContext.OperationBindingContext.HttpContext.RequestServices;
 
-            var actionContext = requestServices.GetRequiredService<IScopedInstance<ActionContext>>().Value;
-            var formatters = requestServices.GetRequiredService<IScopedInstance<ActionBindingContext>>().Value.InputFormatters;
+            var httpContext = bindingContext.OperationBindingContext.HttpContext;
+            var formatters = requestServices
```

Comment:
Is the next part to figure out how to flow formatters through `ModelBindingContext`?